packages/patternfly-4/react-core/src/components/Switch/Switch.tsx

```diff
@@ -1,49 +1,52 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
+import { HTMLProps, FormEvent } from 'react';
```

Comment:
Don't import these and instead use `React.HTMLProps` and `React.FormEvent` to be consistent with other components.